test/support/time.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Urito.Time do
+  def cast!(time) do
```

Comment:
@BlakeWilliams I'd love some feedback and help on https://github.com/seanpdoyle/urito/pull/7/files#r105440611.